Functions/xASL_adm_RemoveLogFilesForRerun.m

```diff
@@ -0,0 +1,49 @@
+function xASL_adm_RemoveLogFilesForRerun(rootDir)
+%xASL_adm_RemoveLogFilesForRerun Removes all log files from xASL directory
```

Comment:
Better call it something else - xASL directory can be confused with the ExploreASL code. Directory with ASL data in Legacy format, ASL data in BIDS? - Not sure which of those two applies.